sandbox/src/mocks/CreateReferralParameters.json

```diff
@@ -0,0 +1,111 @@
+{
```

Comment:
I wonder if we need a folder structure for the mock responses/expected requests too as I guess this folder will get quite busy when multiple endpoints are added. Perhaps a directory for each endpoint with a "requests" and "responses" folder nested underneath?

Reply:
I think that would work too, i am wondering if there are certain examples that we'd like to share through e.g. ReferralRequest etc which will be quite common outputs ? If we are fine with duplication there I'll create a subdirectory for each endpoint as suggested above.

Reply:
Done